internal/magic/text.go

```diff
@@ -13,6 +13,7 @@ import (
 var (
 	// HTML matches a Hypertext Markup Language file.
 	HTML = markup(
+		append([]byte{0xEF, 0xBB, 0xBF}, []byte("<!DOCTYPE HTML")...),
```

Comment:
This line is not needed when BOM is trimmed.

Reply:
done in /e4f5c6c96533c3e35b8bd01f5161308cba48576c